src/main/scala/stage/RocketChipOptions.scala

```diff
@@ -37,5 +40,7 @@ class RocketChipOptions private[stage] (
     case _ =>
       if (!topPackage.isEmpty && !configClass.isEmpty) Some(s"${topPackage.get}.${configClass.get}") else None
   }
+
+  lazy val chiselFileExtension: Option[String] = if (dumpProtobuf) Some(".pb") else Some(".fir")
```

Comment:
```suggestion
  lazy val chiselFileExtension: String = if (dumpProtobuf) ".pb" else ".fir"
```
I don't think there's any reason to make this an `Option` since it will always have a file extension